README.md

```diff
@@ -3,4 +3,38 @@
 A curated list of amazingly awesome resources for (not only) LMC developers.
 
 ## Table of Contents
-To Be Done
+- [Development](#development)
+- [Core review](#code-review)
+- [Q&A](#qa)
+- [Git](#git)
+- [Books](#books)
+- [Newsletter](#newsletters)
+
+## Development
+- [@Ocramius - Fluent interfaces are evil](https://ocramius.github.io/blog/fluent-interfaces-are-evil/)
```

Comment:
What about swap author and article title? The title is more important imo. (The same below)